governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Template - [Executive Vote] Aligned Delegate Compensation, AVC Member Compensation, Aave Revenue Share Payment, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024
+summary: Aligned Delegate and AVC member compensation, Spark Protocol-Aave Share Payment, Whitelisting new address in the RWA015-a output conduit, pushing 84,211.27 USDP out of the MCD PSM PAX input conduit jar, triggering a Spark proxy spell on Ethereum and Gnosis.
+date: 2024-04-18T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] [Executive Vote] Aligned Delegate Compensation, AVC Member Compensation, Aave Revenue Share Payment, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024
+
+The Governance Facilitators and SideStream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- A total of **219.36 MKR** will be distributed to eight Aligned Delegates as compensation for March 2024.
+- A total of **175.14 MKR** will be distributed to nine AVC members as compensation for March 2024.
+- A total of **238,339 DAI** will be paid to Aave as part of the SparkLend revenue share.
+- A new address in the RWA015-A output conduit will be whitelisted.
+- **84,211.27 USDP** will be pushed out of the  MCD PSM PAX Input Conduit Jar.
+- A proxy spell for Spark Protocol on Ethereum and Gnosis will be triggered.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
```

Comment:
```suggestion
Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
```